python/py-pytest-mpl/Portfile

```diff
@@ -1,39 +1,61 @@
 
 PortSystem          1.0
-PortGroup           python 1.0
 PortGroup           github 1.0
+PortGroup           python 1.0
 
-github.setup        matplotlib pytest-mpl 0.9 v
-name                py-${name}
+github.setup        matplotlib pytest-mpl 0.11 v
+revision            0
+name                py-${github.project}
 categories-append   devel
+
 license             BSD
+maintainers         nomaintainer
 platforms           darwin
-supported_archs     noarch
-maintainers         {aronnax @lpsinger} openmaintainer
 
-description         pytest plugin to for testing figures output from Matplotlib
-long_description    This is a plugin to faciliate image comparison for \
-                    Matplotlib figures in pytest. Matplotlib includes a number \
-                    of test utilities and decorators, but these are geared \
-                    towards the nose testing framework. Pytest-mpl makes it \
-                    easy to compare figures produced by tests to reference \
-                    images when using pytest.
+description         pytest plugin to help with testing figures output\
+                    from Matplotlib
+
+long_description    This is a plugin to facilitate image comparison\
+                    for Matplotlib figures in pytest. Matplotlib\
+                    includes a number of test utilities and\
+                    decorators, but these are geared towards the nose\
+                    testing framework. Pytest-mpl makes it easy to\
+                    compare figures produced by tests to reference\
+                    images when using pytest. For each figure to test,\
+                    the reference image is subtracted from the\
+                    generated image, and the RMS of the residual is\
+                    compared to a user-specified tolerance. If the\
+                    residual is too large, the test will fail (this is\
+                    implemented using helper functions from\
+                    matplotlib.testing).
+
+checksums           rmd160  74ffff3c8adb4dca9dd80003b07848a3845c90e4 \
+                    sha256  6f9e176e2256831f96d21f1e2ec19ba53e7ec614b7e4ed4513f088967f7610d9 \
+                    size    193980
+
+python.versions     37 38
 
-python.versions     27 35 36
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools_scm
 
-checksums           rmd160  6b135b3a08453376c88e23f434a1fc901d54bc9d \
-                    sha256  960d02c75d6b3b75043060c0b1fcdf98099ce1b6e72d3b3bd508bc13f074a152 \
-                    size    193158
+    depends_run-append \
+                    port:py${python.version}-matplotlib
 
-if {${name} ne ${subport}} {
-  depends_build-append \
-                    port:py${python.version}-setuptools
+    depends_test-append \
+                    port:py${python.version}-pytest-cov
```

Comment:
I know that pytest-cov is in the extra requirements, but I doubt that it is actually necessary except for coverage measurement.

Reply:
This commit dropped. Please see above.